manifests/instance.pp

```diff
@@ -316,77 +305,48 @@
   }
 
   if $manage_service_file {
-    $service_provider_lookup = pick(getvar('service_provider'), false)
-
-    if $service_provider_lookup == 'systemd' {
-      file { "/etc/systemd/system/${service_name}.service":
-        ensure  => file,
-        owner   => 'root',
-        group   => 'root',
-        mode    => '0644',
-        content => template('redis/service_templates/redis.service.erb'),
-      }
-
-      # Only necessary for Puppet < 6.1.0,
-      # See https://github.com/puppetlabs/puppet/commit/f8d5c60ddb130c6429ff12736bfdb4ae669a9fd4
-      if versioncmp($facts['puppetversion'],'6.1.0') < 0 {
-        include systemd::systemctl::daemon_reload
-        File["/etc/systemd/system/${service_name}.service"] ~> Class['systemd::systemctl::daemon_reload']
-      }
+    file { "/etc/systemd/system/${service_name}.service":
+      ensure  => file,
+      owner   => 'root',
+      group   => 'root',
+      mode    => '0644',
+      content => template('redis/service_templates/redis.service.erb'),
+    }
 
-      if $title != 'default' {
-        service { $service_name:
-          ensure     => $service_ensure,
-          enable     => $service_enable,
-          hasrestart => $service_hasrestart,
-          hasstatus  => $service_hasstatus,
-          subscribe  => [
-            File["/etc/systemd/system/${service_name}.service"],
-            Exec["cp -p ${redis_file_name_orig} ${redis_file_name}"],
-          ],
-        }
-      }
-    } else {
-      file { "/etc/init.d/${service_name}":
-        ensure  => file,
-        mode    => '0755',
-        content => template("redis/service_templates/redis.${facts['os']['family']}.erb"),
-      }
+    # Only necessary for Puppet < 6.1.0,
+    # See https://github.com/puppetlabs/puppet/commit/f8d5c60ddb130c6429ff12736bfdb4ae669a9fd4
+    if versioncmp($facts['puppetversion'],'6.1.0') < 0 {
```

Comment:
+1 for leaving this in for now.  It could catch people out. When we remove it, we could replace it with a note in the README, either with an example people can use in their profiles, or just a quick link to eg. this or another comment.